lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2068,12 +2068,29 @@ def add_contour_set(
                 art3d.line_collection_2d_to_3d(linec, z, zdir=zdir)
 
     def add_contourf_set(self, cset, zdir='z', offset=None):
+        """
```

Comment:
Adding a `levels` return value seems like an ad-hoc addition for a public API. While adding a return value is non-breaking, returning the levels seems somewhat arbitrary.

More generally, it seems like `add_countourf_set` is rather an internal helper. I'm inclined to make this private and add a compatiblity method back in:
```
def add_contourf_set(...):
    self._add_contourf_set(...)
```
We can later deprecate that easily.

Reply:
Done.